mlflow/tracking/_model_registry/fluent.py

```diff
@@ -146,6 +183,36 @@ def _register_model(
     return create_version_response
 
 
+def _get_logged_models_from_run(source_run: str, model_name: str) -> LoggedModel:
+    """Get all logged models from the source rnu that have the specified model name.
+
+    Args:
+        source_run: Source run from which to retrieve logged models.
+        model_name: Name of the model to retrieve.
+    """
+    client = MlflowClient()
+    logged_models, page_token = [], None
```

Comment:
```suggestion
    page_token = None
    logged_models = []
```

to avoid unnecessary tuple creation and unpacking.